R/learner_survival_surv_parametric.R

```diff
@@ -1,54 +1,66 @@
 #' @title Survival Fully Parametric Learner
 #' @name mlr_learners_surv.parametric
-#' @author RaphaelS1
+#' @author bblodfon
 #'
 #' @description
 #' Parametric survival model.
-#' Calls [survival::survreg()] from \CRANpkg{survival}.
+#' Calls [survivalmodels::parametric()] from 'survivalmodels'.
 #'
 #' @section Custom mlr3 parameters:
 #' - `discrete` determines the class of the returned survival probability
 #' distribution. If `FALSE` (default) continuous probability
 #' distributions are returned using [distr6::VectorDistribution], otherwise
-#' [distr6::Matdist].
+#' [distr6::Matdist] (faster to calculate survival measures that require a
+#' `distr` prediction type).
 #'
 #' @template learner
 #' @templateVar id surv.parametric
+#' @template install_survivalmodels
 #'
 #' @details
-#' This learner allows you to choose a distribution and a model form to compose a predicted
-#' survival probability distribution.
+#' This learner allows you to choose a distribution and a model form to compose
+#' a predicted survival probability distribution.
 #'
-#' The internal predict method is implemented in this package as our implementation is more
-#' efficient for composition to distributions than [survival::predict.survreg()].
+#' The predict method is implemented in [survivalmodels::predict.parametric()].
+#' Our implementation is more efficient for composition to distributions than
+#' [survival::predict.survreg()].
 #'
-#' `lp` is predicted using the formula \eqn{lp = X\beta} where \eqn{X} are the variables in the test
-#' data set and \eqn{\beta} are the fitted coefficients.
+#' Three types of prediction are returned for this learner:
```

Comment:
Maybe we want to generate this programmatically, i.e. something like `r surv_predict_types("lp", "crank")`

Reply:
It's a good idea. Since I plan to go through all survival learners and chnage the prediction order (#331), I can see how many need that prediction info and refactor it?

Reply:
see #347

Reply:
or better a doc template?

Reply:
Both is fine, whatever works :)